logstash-core-event-java/src/main/java/com/logstash/bivalues/ObjectBiValue.java

```diff
@@ -0,0 +1,41 @@
+package com.logstash.bivalues;
+
+import org.jruby.Ruby;
+import org.jruby.javasupport.JavaUtil;
+import org.jruby.runtime.builtin.IRubyObject;
+
+import java.io.ObjectStreamException;
+
+// Guy: I am not sure we need this class
+// it was meant to be supplied as a fall-back when BiValues.newBiValue(obj) returns nothing.
+// i.e. when JRuby wraps a value in a JavaProxy, however, we don't want Maps, Lists or arrays in a BiValue
+// or when there is a JRuby class that has no java equivalent, for example, a custom class.
+// in the case of a ConcreteProxy, toJava(Object.class) will return the wrapped Java object.
```

Comment:
so do we trash it? doesn't that defeat the purpose of not accepting "unknown" objects?